team_datasnokmennene/index.md

```diff
@@ -0,0 +1,53 @@
+# American Cookies
+### By Datasnokmennene
+
+```python
+import drawer
+import fridge
+import time
+import appliances	
+
+def mix_butter_and_sugar(butter, sugar):
+    mix = appliances.mix(butter, sugar)
+    mix.append(vanilla*0.05)
```

Comment:
what units are these?

Reply:
I fixed the units. Please review again :D